### ISSUE TITLE
Big Crash

### ISSUE THREAD

**Issue by Tuvqlu** [FlxiBoy1313/Perks#37]
Using Dev 87, when i buy a perk for myself (speed), this happens:

TypeError: "str_replace(): Argument #2 ($replace) must be of type array|string, int given" (EXCEPTION) in "plugins/Perks_dev-87.phar/src/flxiboy/Perks/form/PerkForm" at line 149

Help please :(

**Comment by Tuvqlu** [comment]
Also setting it in certain worlds does not work, it disables it for all

**Comment by FlxiBoy1313** [comment]
Okay thx, this bug will in next released fixed.

**Comment by Tuvqlu** [comment]
Good because it’s a big huge problem! Hope to see the fix soon! Bc I want some for myself haha

**Comment by PocketMiner92** [comment]
Hope to see the Fix soon!
You should learn to say "Thanks"

**Comment by Tuvqlu** [comment]
Yes of course, sorry. Thankyou so much for the quick reply xd. Thanks PocketMiner

**Comment by Tuvqlu** [comment]
Yeah it’s really great uploading a plugin to pocketmine when you could use it privately instead. Developers are so great and so helpful and make so many servers popular so thankyou so much y’all !

**Comment by PocketMiner92** [comment]
You should use Powernukkit.
Its much better with Performance, more Plugins, better worlds with Vanilla Plugins and 1.16 Blocks.
Much More Features as Pocketmine have

**Comment by Tuvqlu** [comment]
I’m so deep into pocketmine I can’t get out now, LOL

**Comment by PocketMiner92** [comment]
Oh, Hmmm
Have fun with Pocketmine.
🤣🤣🤣🤣

**Comment by Tuvqlu** [comment]
Lol wanna know smth funny.

On my server I kept gifting people perks and then I decided I should treat myself to one, bc no one bought me any. I guess they spend their coins on other things :(. So I press the button and the server explodes. TnT flies and the server crashes. Then the whole internet shuts down, then the world. Turns out the world is actually a server, a very advanced one. The owner of that server sees what cashed and it says

Minecraft Plugin

lol
